static/js/vue-cdr-access/src/components/listDisplay.vue

```diff
@@ -100,7 +100,7 @@ Renders search results in a list view display format
             },
 
             showCollection(record) {
-                return record.type !== 'AdminUnit' && record.type !== 'Collection' &&
+                return record.type !== 'AdminUnit' && record.type === 'Collection' &&
```

Comment:
I think this means that only Collection objects would pass now. I think your goal here is to get it to also show the archival collection id for Collection objects? If so, then I think removing `&& record.type === 'Collection'` would have that effect